qiskit_ibm_runtime/accounts/storage.py

```diff
@@ -40,6 +42,26 @@ def save_config(filename: str, name: str, config: dict, overwrite: bool) -> None
         json.dump(data, json_out, sort_keys=True, indent=4)
 
 
+def read_qiskitrc(qiskitrc_config_file: str, default_config_file: str) -> None:
+    """Save qiskitrc credentials into qiskit-ibm.json config file."""
+    config_parser = ConfigParser()
+    config_parser.read(qiskitrc_config_file)
+    for name in config_parser.sections():
+        account_data = dict(config_parser.items(name))
+    save_config(
+        filename=default_config_file,
+        name="default-ibm-quantum",
+        overwrite=True,
+        config=Account(
+            token=account_data["token"],
+            url=account_data["url"],
+            channel="ibm_quantum",
+        )
+        .validate()
+        .to_saved_format(),
+    )
```

Comment:
Saving should not be done from this storage file, read from the file here and then return the read data and then from management file call save_config to save the data.